tests/test_types_zoneinfo.py

```diff
@@ -48,3 +51,8 @@ def test_zoneinfo_json_schema():
         'properties': {'tz': {'type': 'string', 'format': 'zoneinfo', 'title': 'Tz'}},
         'required': ['tz'],
     }
+
+
+def test_zoneinfo_union() -> None:
+    ta = TypeAdapter(Union[zoneinfo.ZoneInfo, timezone], config=ConfigDict(arbitrary_types_allowed=True))
```

Comment:
What's the reason to have `arbitrary_types_allowed` here?

Reply:
We don't support `timezone` 😅